src/FsCheck.NUnit/paket.template

```diff
@@ -1,31 +0,0 @@
-type file
-id
-    FsCheck.NUnit
-authors
-    Kurt Schelfthout and contributors
-owners
-    Kurt Schelfthout and contributors
-projectUrl
-    https://github.com/fsharp/FsCheck
-iconUrl
-    https://raw.githubusercontent.com/fscheck/FsCheck/master/docs/files/img/logo.png
-licenseUrl
-    https://github.com/fsharp/FsCheck/blob/master/License.txt
-requireLicenseAcceptance
-    false
-copyright
-    Copyright 2017
-tags
-    test testing random fscheck quickcheck nunit
-summary
-    Integrates FsCheck with NUnit
```

Comment:
Note that we have lost the `Summary` field. I couldn't find an equivalent, and indeed I couldn't see this text on NuGet.org at all.

Reply:
totally fine, no need to spend time on such things